### PR TITLE
Upgrade test document updates

### PR DESCRIPTION
**Pull request by Jaakko-Os**
https://github.com/metal3-io/metal3-dev-env/pull/453 to be merged first

### CODE REVIEW

File: scripts/feature_tests/README.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 [![Ubuntu V1alpha3 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_ubuntu/badge/icon?subject=Feature-tests)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_ubuntu/)
```
Comment (furkatgofurov7) [inline scripts/feature_tests/README.md:5]:
```suggestion
[![Ubuntu V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_ubuntu/badge/icon?subject=Feature-tests)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_ubuntu/)
```

File: scripts/feature_tests/README.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
...
 [![Centos V1alpha3 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_centos/badge/icon?subject=Feature-tests-centos)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_centos/)
 [![Ubuntu V1alpha3 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_ubuntu/badge/icon?subject=Feature-tests-upgrade)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_ubuntu/)
+[![Ubuntu V1alpha3 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_centos/badge/icon?subject=Feature-tests-upgrade-centos)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_centos/)
```
Comment (furkatgofurov7) [inline scripts/feature_tests/README.md:8]:
```suggestion
[![Centos V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_centos/badge/icon?subject=Feature-tests-upgrade-centos)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_centos/)
```

File: scripts/feature_tests/README.md [hunk 3]
```diff
@@ -5,6 +5,7 @@
 [![Ubuntu V1alpha3 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_ubuntu/badge/icon?subject=Feature-tests)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_ubuntu/)
 [![Centos V1alpha3 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_centos/badge/icon?subject=Feature-tests-centos)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_centos/)
```
Comment (furkatgofurov7) [inline scripts/feature_tests/README.md:6]:
```suggestion
[![Centos V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_centos/badge/icon?subject=Feature-tests-centos)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_centos/)
```

File: scripts/feature_tests/README.md [hunk 4]
```diff
@@ -5,6 +5,7 @@
...
 [![Ubuntu V1alpha3 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_ubuntu/badge/icon?subject=Feature-tests-upgrade)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_ubuntu/)
```
Comment (furkatgofurov7) [inline scripts/feature_tests/README.md:7]:
```suggestion
[![Ubuntu V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_ubuntu/badge/icon?subject=Feature-tests-upgrade)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_ubuntu/)
```

**Review by furkatgofurov7 — COMMENTED** [review]
Looks good, some small nits.

File: scripts/feature_tests/README.md (hunk 1 above)
Comment (Jaakko-Os) [reply scripts/feature_tests/README.md]: done

File: scripts/feature_tests/README.md (hunk 2 above)
Comment (Jaakko-Os) [reply scripts/feature_tests/README.md]: done

File: scripts/feature_tests/README.md (hunk 3 above)
Comment (Jaakko-Os) [reply scripts/feature_tests/README.md]: done

File: scripts/feature_tests/README.md (hunk 4 above)
Comment (Jaakko-Os) [reply scripts/feature_tests/README.md]: done

**Review by furkatgofurov7 — COMMENTED** [review]
lgtm

**Comment by furkatgofurov7** [comment]
We can trigger the test, when CI is back running.

File: scripts/feature_tests/README.md [hunk 5]
```diff
@@ -117,14 +126,6 @@ export CONTAINER_RUNTIME=podman
 export NUM_NODES=4
 ```
 
-Environment variables needed for upgrade tests running clusterctl,
```
Comment (furkatgofurov7) [inline scripts/feature_tests/README.md:120]: By the way, why are removing these variables?

File: scripts/feature_tests/README.md [hunk 6]
```diff
@@ -117,14 +126,6 @@ export CONTAINER_RUNTIME=podman
...
-set as default values:
-
-```bash
-export CAPM3RELEASE=v0.3.2
```
Comment (furkatgofurov7) [inline scripts/feature_tests/README.md:124]: By the way, why are removing these variables?

Comment (Jaakko-Os) [reply scripts/feature_tests/README.md:124]: Please see above explanation.

**Pull request merged**